tests/baselines/reference/intersectionSatisfiesConstraint.errors.txt

```diff
@@ -0,0 +1,23 @@
+intersectionSatisfiesConstraint.ts(10,60): error TS2550: Property 'assign' does not exist on type 'ObjectConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2015' or later.
+
+
+==== intersectionSatisfiesConstraint.ts (1 errors) ====
+    interface FirstInterface {
+        commonProperty: number
+    }
+    
+    interface SecondInterface {
+        commonProperty: number
+    }
+    
+    const myFirstFunction = <T extends FirstInterface | SecondInterface>(param1: T) => {
+        const newParam: T & { otherProperty: number } = Object.assign(param1, { otherProperty: 3 })
+                                                               ~~~~~~
+!!! error TS2550: Property 'assign' does not exist on type 'ObjectConstructor'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2015' or later.

```

Comment:
it would be nice to have 0 errors by switching to es2015 or higher, or else by moving `newParam` to being a parameter so it doesn't have an initialiser.

Reply:
Oops, my bad.